universal-login-relayer/lib/utils/pendingExecution.ts

```diff
@@ -0,0 +1,71 @@
+import WalletContract from '@universal-login/contracts/build/Wallet.json';
+import {utils, ContractFactory, Contract, Wallet} from 'ethers';
+import {sortExecutionsByKey} from '../utils/utils';
+import {concatenateSignatures} from '@universal-login/contracts';
+import {Message} from '@universal-login/commons';
+
+export default class PendingExecution {
```

Comment:
I would propose to split this class into two:
`PendingExecution` and `PendingExecutionService`.

`PendingExecution` is pure data, while `PendingExecutionService` is only functions. It will make it easier to test and introduce database later along the way.

Reply:
so i have to create another service and import it into WalletService?